interactions/client/bot.pyi

```diff
@@ -97,6 +98,11 @@ class Client:
     def reload(
         self, name: str, package: Optional[str] = None, *args, **kwargs
     ) -> Optional["Extension"]: ...
+    async def modify(
+        self,
+        username: Optional[str] = MISSING,
+        avatar: Optional[Image] = MISSING,
+    ) -> User: ...
```

Comment:
I think this should be moved underneath `get_extension()` because this function is basically in between two functions related to `Extension`s.